app/assets/json/translations/locale-sj-en.json

```diff
@@ -7,6 +7,6 @@
       "FEEDBACK_URL": "https://forms.gle/fBpnRoM6XZJRxmxR6"
     },
     "T": {
-      "DAHLIA_HOUSING_PORTAL": "<strong>DAHLIA</strong><br/>San Jose"
+      "DAHLIA_HOUSING_PORTAL": "<strong>DAHLIA</strong><br/><span class=\"group\">San Jose</span>"
```

Comment:
@jaredcwhite mind if i update the "group" class name here and in tests to be slightly more specific, e.g. "header-logo-group"?

Reply:
@callaghanc Sure, sounds good